src/parser/SparqlParser.h

```diff
@@ -14,7 +14,10 @@
 class SparqlParser {
  public:
   static ParsedQuery parseQuery(std::string query);
-  // A convenience function for parsing the query and setting the datasets.
-  static ParsedQuery parseQuery(std::string operation,
+  static std::vector<ParsedQuery> parseUpdate(std::string update);
+  // Convenience functions for parsing the operation and setting the datasets.
+  static ParsedQuery parseQuery(std::string query,
                                 const std::vector<DatasetClause>& datasets);
+  static std::vector<ParsedQuery> parseUpdate(
+      std::string update, const std::vector<DatasetClause>& datasets);
```

Comment:
Now these need more documentation (especially the separate overloads with the explicit datasets). But maybe that is useful to unify once we have merged the 1845, where not much is missing.

Reply:
Yes, that will get much nice after 1845. Both parsing functions should then be as simple as the overloads without datasets are now.